src/main/java/io/vertx/core/impl/EventbusAddresses.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.vertx.core.impl;
+
+import java.util.UUID;
+
+/**
+ * @author Joel Takvorian
+ */
+public final class EventbusAddresses {
+  private EventbusAddresses() {
+  }
+
+  public static String generateInternal(String module) {
```

Comment:
move it to another class (like MessageImpl), no need to create an extra class just for this purpose